lib/register_common/services/set_client_redis.rb

```diff
@@ -0,0 +1,42 @@
+module RegisterCommon
+  module Services
+    class SetClientRedis
```

Comment:
This seems like quite a lot of work to wrap the set handling Redis already has, especially given that the tests still end up mocking `set_client` anyway. How likely is it that this additional level of abstraction will be useful in practice, do you think?

Reply:
Discussed.